### DISCUSSION TITLE
23.05 首页不显示架构信息

### DISCUSSION THREAD

**Discussion by nantayo** [immortalwrt/immortalwrt#1157] [Q&A]
各位，我使用的是自己编译的 23.05-SNAPSHOT 分支固件，2天前我升级到 r0-4245e51 版本后，发现首页概览无法显示架构信息，一台 X86 架构的小主机和一台 armsr/armv8 架构的斐讯 N1（S905D）均无法显示，情况见下图：
![1](https://github.com/immortalwrt/immortalwrt/assets/77558823/d53bb0e7-bdef-4f8e-baf0-9467b8f78799)
![2](https://github.com/immortalwrt/immortalwrt/assets/77558823/c37a63f2-fc38-4a76-8947-1f4cee906e85)
***
已确认固件已安装了 autocore、sensors 等包，ssh 连接后也可以通过相关命令读取温度等信息；已确认恢复到出厂设置无效。
对于这种情况，有人知道可能的问题原因吗？简要翻看了一下 immortalwrt 和 luci 这两个库近期的提交记录，不知是否与 autocore 这个包的更新有关？

**Comment by 1715173329** [comment]
修了

**Reply by nantayo** [reply]
收到，现在拉取新源码试一下

**Comment by nantayo** [comment]
确认最新源码已修复该问题
